qdarkstyle/__init__.py

```diff
@@ -278,6 +283,8 @@ def _load_stylesheet(qt_api='', palette=None):
         palette = LightPalette
     else:
         print("Not recognized ID for palette! Exiting!")
+        print("Only 'dark' and 'light' palette IDs are supported by load_stylesheet().")
```

Comment:
I would say the limitation here is more like:

```suggestion
        print("Only 'DarkPalette' and 'LightPalette' palettes (with 'dark' and 'light' IDs respectively) are supported by load_stylesheet().")
```